app/javascript/controllers/projects/samples/metadata/delete_listing_controller.js

```diff
@@ -0,0 +1,26 @@
+import { Controller } from "@hotwired/stimulus";
+
+// creates a table listing all selected metadata for deletion
+export default class extends Controller {
+    static targets = ["table"];
+
+    connect() {
+        const body = document.getElementById("metadata-table-body");
+
+        for (let row of body.rows) {
+            const isChecked = row.cells[0].children[0].checked;
+
+            if (isChecked) {
+                const newRow = this.tableTarget.insertRow(-1);
```

Comment:
You are inserting theis row into the `thead`.  Add a `tbody` to your table and update the target to that.

Reply:
Great catch, fixed in b9073c70ebecdaae80f23d5cbe4a5a050279ee7e